sql/maplestory.sql

```diff
@@ -81,6 +81,8 @@ CREATE TABLE `characters` (
   `miniGamePoints` int(11) NOT NULL DEFAULT '2000',
   `buddyListSize` tinyint(3) unsigned NOT NULL DEFAULT '20',
   `inCashShop` int(11) NOT NULL DEFAULT '0',
+  `regTeleportRocks` text NOT NULL,
```

Comment:
I don't think it really matters if this is null, because the logic in-game just set's it to invalidmap by default. Also, because of NOT NULL if we create a new character then it'll crash the login server because it doesn't account for this.